src/vue/common/Pagination.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="pagination">
+    <slot />
+    <button
+      class="pagination-btn"
+      @click="$emit(PAGINATION_EVENTS.loadNextPage)">
+      {{ 'pagination.btn-label-more' | globalize }}
+    </button>
+  </div>
+</template>
+
+<script>
+import { ErrorHandler } from '@/js/helpers/error-handler'
+const PAGINATION_EVENTS = {
```

Comment:
have you read this before?
https://app.asana.com/0/894507180698157/898572251826265